images/router/egress/squid.conf

```diff
@@ -0,0 +1,9 @@
+# egress-router squid.conf
+# This is a mostly-transparent HTTP/HTTPS proxy; it only exists to
+# redirect requests, not to authenticate them in any way.
+
+http_port 8080
+http_access allow all
+cache deny all
+access_log none all
+debug_options ALL,0
```

Comment:
It's not the end of the world.  But you could try `/proc/self/fd/2` if the message really annoyed you.  I'd probably just live with that error though.